kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -184,9 +192,10 @@ public KyuubiConnection(String uri, Properties info) throws SQLException {
           client = new TCLIService.Client(new TBinaryProtocol(transport));
           // open client session
           openSession();
-          getLaunchEngineLog();
-          executeInitSql();
-
+          if (!isBeeLineMode) {
+            showLaunchEngineLog();
+            executeInitSql();
```

Comment:
Why not run `executeInitSql()` in beeline mode?

Reply:
it will block, we need launch engine log thread at first, then exeuteInitSql